app/Rest/Employees_Controller.php

```diff
@@ -0,0 +1,431 @@
+
+		$query_params['exclude'] = array(
+			'description' => __( 'Ensure result set excludes specific IDs.' ),
+			'type'        => 'array',
+			'items'       => array(
+				'type' => 'integer',
+			),
+			'default'     => array(),
+		);
+
+		$query_params['include'] = array(
+			'description' => __( 'Limit result set to specific IDs.' ),
+			'type'        => 'array',
+			'items'       => array(
+				'type' => 'integer',
+			),
+			'default'     => array(),
+		);
+
+		$query_params['offset'] = array(
+			'description' => __( 'Offset the result set by a specific number of items.' ),
+			'type'        => 'integer',
+		);
+
+		$query_params['order'] = array(
+			'description' => __( 'Order sort attribute ascending or descending.' ),
+			'type'        => 'string',
+			'default'     => 'desc',
+			'enum'        => array( 'asc', 'desc' ),
+		);
+
+		$query_params['orderby'] = array(
+			'description' => __( 'Sort collection by object attribute.' ),
+			'type'        => 'string',
+			'default'     => 'date',
+			'enum'        => array(
+				'author',
+				'date',
+				'id',
+				'include',
+				'modified',
+				'parent',
+				'relevance',
+				'slug',
+				'include_slugs',
+				'title',
+			),
+		);
+
+		return $query_params;
+	}
+
+	/**
+	 * Retrieves the item's schema, conforming to JSON Schema.
+	 *
+	 * @return array
+	 */
+	public function get_item_schema() {
+		$schema = array(
+			'$schema'    => 'http://json-schema.org/draft-04/schema#',
+			'title'      => Employee_Post_Type::$ID,
+			'type'       => 'object',
+			'properties' => array(
+				'title'         => array(
+					'description' => sprintf( __( 'The title for the %s.' ), Employee_Post_Type::$ID ),
+					'type'        => 'string',
+				),
+				'content'       => array(
+					'description' => sprintf( __( 'The content for the %s.' ), Employee_Post_Type::$ID ),
+					'type'        => 'string',
+				),
+				'custom_fields' => array(
+					'description' => __( 'Custom fields.' ),
+					'type'        => 'object',
+					'properties'  => array(
+						'level'    => array(
+							'description' => sprintf( __( 'The level for the %s.' ), Employee_Post_Type::$ID ),
+							'type'        => 'string',
+							'enum'        => array(
+								'Junior',
+								'Middle',
+								'Senior',
+							),
+						),
+						'position' => array(
+							'description' => sprintf( __( 'The position for the %s.' ), Employee_Post_Type::$ID ),
+							'type'        => 'string',
+						),
+						'bio'      => array(
+							'description' => sprintf( __( 'The bio for the %s.' ), Employee_Post_Type::$ID ),
+							'type'        => 'string',
+						),
+					),
+				),
+			),
+		);
+
+		return $this->add_additional_fields_schema( $schema );
+	}
+
+	/**
+	 * Create_update_employee
+	 *
+	 * @param object $prepared_post Prepared post object.
+	 * @param string $type Action type.
+	 *
+	 * @return int|\WP_Error
+	 */
+	protected function create_update_employee( $prepared_post, $type = 'create' ) {
```

Comment:
please remove "type", you can define create or update by "id". if no ID - this is create.